Mathlib/CategoryTheory/Sites/EffectiveEpimorphic.lean

```diff
@@ -298,6 +298,15 @@ attribute [nolint simpNF]
   EffectiveEpiFamily.fac
   EffectiveEpiFamily.fac_assoc
 
+/-- The effective epi family structure on the identity -/
+def effectiveEpiFamilyStructId : EffectiveEpiFamilyStruct (α : Unit → C) (fun _ => 𝟙 (α ())) where
```

Comment:
This has strange formulation: is there a reason to write `(α : Unit → C)` instead of `(c : :C)`?

Reply:
Ops, yes I meant `(c : C)`. I understand that at some point you need to turn an element of `C` to a morphism from `Unit`. I am just wondering if one someone will use your `def` they'll already have the morphism or rather the element. It's just to make this definition more comfortable to use.

Reply:
I thought a bit about it and I think one would need to introduce a bit more API to generalize this (such as `EffectiveEpiFamilies` being invariant under composing by bijections of the indexing set).

Reply:
Í'll leave it as is for now.